Library/Homebrew/test/cask/artifact/binary_spec.rb

```diff
@@ -70,11 +70,13 @@
   it "clobbers an existing symlink" do
```

Comment:
“does not …”

Reply:
or “avoids clobbering …” to match the test above this one

Reply:
Fixed, thank you @reitermarkus and @SeekingMeaning.

Stupid oversight on my part.